recipes-soletta/soletta/soletta/config

```diff
@@ -1,92 +1,147 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# soletta configuration
+# Soletta configuration
 #
-#HAVE_UDEV=y
-HAVE_SYS_AUXV_H=y
-# HAVE_DECL_STRNDUPA is not set
+HAVE_ACCEPT4=y
+HAVE_BUILTIN_ADD_OVERFLOW=n
+HAVE_BUILTIN_MUL_OVERFLOW=n
+HAVE_CHECK=y
+# HAVE_CONTIKI is not set
+HAVE_DECL_DLADDR=y
+HAVE_DECL_IFLA_INET6_ADDR_GEN_MODE=y
+HAVE_DECL_IFLA_INET6_MAX=y
+HAVE_DECL_MEMMEM=y
+HAVE_DECL_STRNDUPA=y
+HAVE_DLFCN_H=y
+# HAVE_DUKTAPE_SRC is not set
+HAVE_FLTO=y
 HAVE_GLIB=y
-HAVE_PTHREAD_H=y
+HAVE_GNU_SOURCE=y
 HAVE_GTK=n
-# HAVE_DECL_IFLA_INET6_ADDR_GEN_MODE is not set
-HAVE_DLFCN_H=y
-HAVE_PPOLL=y
+HAVE_ICU=y
+HAVE_ISATTY=y
+HAVE_LDL=y
+HAVE_LIBCURL=y
+HAVE_LIB_MATH=y
 HAVE_LINUX=y
-HAVE_CHECK=y
+HAVE_LOCALE=y
+HAVE_NEWLOCALE=y
+HAVE_NO_SANITIZE=y
+HAVE_PPOLL=y
+HAVE_PTHREAD_H=y
 # HAVE_RIOTOS is not set
-HAVE_ISATTY=y
-# HAVE_SYSTEMD is not set
-HAVE_ACCEPT4=y
-# HAVE_KDBUS is not set
+#HAVE_SANITIZE_ADDRESS is not set
+#HAVE_SANITIZE_UNDEFINED is not set
+HAVE_SD_DBUS=y
+HAVE_SOCKET=y
+HAVE_STRTOD_L=y
+HAVE_SYSTEMD=y
+HAVE_SYS_AUXV_H=y
+HAVE_UDEV=y
+FEATURE_DYNAMIC_LINKER=y
+FEATURE_RUNNABLE_PROGRAMS=y
+FEATURE_WORKER_THREADS=y
+FEATURE_UNIX_SOCKETS=y
+FEATURE_NETWORK=y
+FEATURE_COAP=y
+FEATURE_HTTP_CLIENT=y
+FEATURE_HW_AIO=y
+FEATURE_HW_PWM=y
+FEATURE_HW_GPIO=y
+FEATURE_HW_UART=y
+FEATURE_HW_SPI=y
+FEATURE_HW_I2C=y
+FEATURE_FLOW=y
+FEATURE_FILESYSTEM=y
+FEATURE_CC_SANITIZE=y
+BASE_OS="linux"
+LINUX=y
 
```

Comment:
none of the above should be here. They must be detected when doing `make alldefconfig` (or friends).